integration-tests/src/test/java/tachyon/security/MasterClientAuthenticationIntegrationTest.java

```diff
@@ -161,6 +161,8 @@ public void customAuthenticationDenyConnectTest() throws Exception {
             mLocalTachyonCluster.getMasterTachyonConf());
     Assert.assertFalse(masterClient.isConnected());
     masterClient.connect();
+    // should never reach here, but this silences the resource not closed warning
+    masterClient.close();
```

Comment:
out of curiosity, is there an annotation that could be used with the same effect?